### ISSUE TITLE
Single Reply to Group Message in wrong conversation

### ISSUE THREAD

**Issue by bokf** [deltachat/deltachat-core#660]
I noticed following behavior, when I send a message to a group and somebody answers to my message, excluding the other group members, the message is shown in the group conversation anyway. According to my understanding a single reply to a group message should go to the single conversation with that person and not to the group conversation. In DeltaChat I can not see, if the email has been answered to the whole group or if it is only a personal mail to me.

**Comment by testbird** [comment]
I agree that direct, one-to-one replies to group messages should go into the one-to-one chat. That is to make the direct reply obvious in a chat program. All one-to-one replies should always be displayed in the one-to-one chat.

In general, the situation seems also to be clear **when extending** a group, or multi-recipient email thread. No matter if new recipients are added by deltachat clients or classic MUA users, _replies_  should not create a new group but extent the group member list.


When **recipients are reduced** there could be a better compromise, handling email and chat clients differently.
As chat client users are used to think of "group entities" and managing group members before replying, the existing group gets modified on all clients.

Classic MUA usage, however, is more flexible and some users are used to create parallel threads (chats) by reducing the recipients in a reply. So to represent this in a chat client would mean to create a new, parallel chat to display a reply with reduced recipients.

The new thread's chat allows chat client users to reply and react as appropriate.
Possibly answering in the new thread. However, a button or "confirm removal of X from group?" prompt to alternatively keep the larger group's history and reduce the group size instead could also be a very useful option here.

**Comment by csb0730** [comment]
Difficult situation,
 but most of time I saw this problem someone with classic MUA replied to me only ("reply" button) and not to all group members ("reply all" button). This is the standard failure situation with classic MUA but fully user dependent. Some users don't think about the difference. That's my experience.

Maybe we should route such a mail to the sourcing group chat but show an indicator, that this mail is not addressed to all group members?

**Comment by testbird** [comment]
Right, a mistaken reply to sender is a common mistake with some MUAs. An indicatoion would be very good.

I guess though, that an errornous one-to-one reply should still (correctly) appear in the one-to-one chat, however, with an indication that it was send `in-reply-to` a group message. (Some sort of message linking icons? may be necessary for the mailinglist support, anyway.)

**Comment by Simon-Laux** [comment]
sounds like it could also be https://github.com/deltachat/deltachat-core-rust/issues/163 in some cases
